e2e/atlas/clusters_test.go

```diff
@@ -76,6 +76,25 @@ func TestClustersFlags(t *testing.T) {
 		a.Contains(string(resp), "Cluster available")
 	})
 
+	t.Run("LoadSampleData", func(t *testing.T) {
```

Comment:
[q] this may take a while to complete, are you sure you'll be able to delete a cluster while this job is running?
I have scheduled the cluster e2e tests in the PR suite

Reply:
yer you can. I tested it both with UI and mongocli on cloud-dev